### PR TITLE
feat: gnosis activation proposal

### PR DESCRIPTION
**Pull request by brotherlymite**
https://github.com/bgd-labs/aave-proposals-v3/compare/main..feat/gnosis

Deployed payload: https://gnosisscan.io/address/0x6Cf806CF51749bE514B7055b6fDBe346aee47B7b

### CODE REVIEW

**Comment by eboadom** [comment]
Better to abstract this in a function receiving asset and amount https://github.com/bgd-labs/aave-proposals-v3/blob/0f1893e08470cc7145d9680ee3ceaa4a6af1180d/src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3Gnosis_AaveV3GnosisActivation_20231026.sol#L34

Also, should safeApprove/forceApprove be used?

**Comment by eboadom** [comment]
Is the name with or without `-` in the other networks? https://github.com/bgd-labs/aave-proposals-v3/blob/0f1893e08470cc7145d9680ee3ceaa4a6af1180d/src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3Gnosis_AaveV3GnosisActivation_20231026.sol#L71

**Comment by brotherlymite** [comment]
> Better to abstract this in a function receiving asset and amount
> 
> https://github.com/bgd-labs/aave-proposals-v3/blob/0f1893e08470cc7145d9680ee3ceaa4a6af1180d/src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3Gnosis_AaveV3GnosisActivation_20231026.sol#L34
> 
> Also, should safeApprove/forceApprove be used?

Yes, will fix

File: generator/utils/importsResolver.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const GovernanceImports = [
 ] as const;
 
 /**
- * @dev matches the code fro known address book imports and generates an import statment satisfying the used libraries
+ * @dev matches the code fro known address book imports and generates an import statement satisfying the used libraries
```
Comment (sendra) [inline generator/utils/importsResolver.ts:13]:
```suggestion
 * @dev matches the code from known address book imports and generates an import statement satisfying the used libraries
```

File: src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: "Aave v3 Gnosis Activation"
+author: "BGD Labs"
```
Comment (eboadom) [inline src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md:3]:
```suggestion
author: "BGD Labs @bgdlabs"
```

File: src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+- Positive technical evaluation done by BGD Labs of the Gnosis chain network, as described in the [forum](https://governance.aave.com/t/bgd-aave-gnosischain-infrastructure-technical-evaluation/14915) in detail.
+- Positive risk analysis and assets/parameters recommendation by the service providers ACI, Gauntlet and Chaos Labs as commented on the forum.
+- Additionally listing sDAI as approved separately on the [forum](https://governance.aave.com/t/arfc-sdai-onboarding-on-aave-v3-gnosis-pool/15111) and [snapshot](https://snapshot.org/#/aave.eth/proposal/0x31703294f1129b579e98f9384fc95ad39e13eafca127f036d0cc20e927d98f56).
+
+## Specification
+
+The proposal will do the following:
+
+- Set risk steward and freezing steward as the risk admin by executing ACL_MANAGER.addRiskAdmin().
+- Set the guardian address as the pool admin by executing ACL_MANAGER.addPoolAdmin().
```
Comment (eboadom) [inline src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md:25]:
```suggestion
- Set the guardian address as the pool admin by executing ACL_MANAGER.addPoolAdmin(). This is following the standard procedure of keeping pool admin on the Aave Guardian during the bootstrap period, for security.
```

File: src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Aave v3 Gnosis Activation"
+author: "BGD Labs @bgdlabs"
+discussions: "https://governance.aave.com/t/arfc-aave-v3-deployment-on-gnosischain/14695"
+---
+
+## Simple Summary
+
+This proposal allows the Aave governance to activate the Aave V3 Gnosis pool (3.0.2) by completing all the initial setup and listing WETH, wstETH, GNO, USDC, wXDAI, EURe and sDAI as suggested by the service providers (ACI, Gauntlet and Chaos Labs) on the [governance forum](https://governance.aave.com/t/arfc-aave-v3-deployment-on-gnosischain/14695). All the Aave Gnosis V3 addresses can be found in the [aave-address-book](https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Gnosis.sol).
```
Comment (eboadom) [inline src/20231026_AaveV3Gnosis_AaveV3GnosisActivation/AaveV3GnosisActivation.md:9]:
```suggestion
This proposal allows the Aave governance to activate the Aave V3 Gnosis pool (3.0.2) by completing all the initial setup and listing WETH, wstETH, GNO, USDC, wXDAI, EURe and sDAI as suggested by the service providers (ACI, Gauntlet and Chaos Labs) on the [governance forum](https://governance.aave.com/t/arfc-aave-v3-deployment-on-gnosischain/14695/7). All the Aave Gnosis V3 addresses can be found in the [aave-address-book](https://github.com/bgd-labs/aave-address-book/blob/main/src/AaveV3Gnosis.sol).
```

**Review by eboadom — DISMISSED** [review]

**Review by sakulstra — APPROVED** [review]

**Review by eboadom — APPROVED** [review]

**Pull request merged**